indexFormat/implementation.md

```diff
@@ -1,77 +1,67 @@
 
-With Rich Code Navigation, users can use navigation features on their pull requests, without requiring a local checkout. In this scenario, the navigation features are powered through an external service. This service uses a pre-built LSIF index. The index can be generated at a variety of places, like for instance the CI pipeline of the repo:
+1. User creates a new PR.
+1. The CI configured on the repo builds the PR.
+1. The LSIF exporter runs on CI and generates the LSIF index.
 
-1. User creates a pull request
-1. The CI tool builds this pull request
-1. The LSIF exporter runs in the CI pipeline and generates the LSIF file
-
-## LSIF exporter implementations
+## LSIF exporters
 
 | Language | Repository |
 |--|--|
 | TypeScript/JavaScript | [lsif-node](https://github.com/Microsoft/lsif-node) |
+| Java | |
+| C# | |
 
-> Are we missing an implementation? Create a new issue on GitHub.
+> Are we missing an implementation? File a new issue on GitHub to add it here.
 
-## LSIF implementation skeleton
+## LSIF exporter skeleton
 
-As detailed in the specification, the LSIF exporter consists of two tools: the index exporter and the package manager linker.
+As [detailed in the spec](specification.md#project-exports-and-external-imports), the LSIF exporter consists of two tools: the index exporter and the package linker.
 
 ### Index exporter
 
 This tool generates an LSIF dump for a workspace by traversing through source files and storing LSP responses. For TypeScript/JavaScript, [`lsif-tsc`](https://github.com/Microsoft/lsif-node/tree/master/tsc) is the index exporter.
 
-### Package manager linker
+### Package linker
 
-This tool ingests the LSIF output from an index exporter and converts `moniker` vertices into the package manager scheme. Additionally, it links these monikers to `packageInformation` vertices. For TypeScript/JavaScript, [`lsif-npm`](https://github.com/Microsoft/lsif-node/tree/master/npm) is the package manager linker for NPM.
+This tool ingests the LSIF output from an index exporter and converts `moniker` vertices into the package manager scheme. Additionally, it links these monikers to `packageInformation` vertices that reference packages hosted on a registry. For TypeScript/JavaScript, [`lsif-npm`](https://github.com/Microsoft/lsif-node/tree/master/npm) is the package manager linker for NPM.
 
 ## Testing and validation
 
-### lsif-util
+### LSIF validation utility
 
-The `lsif-util` tool can validate your generated LSIF output. Additionally, the tool can also help you comprehend the generated output by visualizing and searching the vertices and edges.
+The [`lsif-util`](https://github.com/jumattos/lsif-util) tool can validate your generated LSIF output. Additionally, the tool can also help you comprehend the generated output by visualizing and searching the vertices and edges.
```

Comment:
Nit: saying "the tool can help you comprehend" comes across condescending. I would just say, "the tool also provides a way to easily search the output and visualization via Viz.js and Graphviz."